packages/space-opera/src/components/ibl_selector/ibl_selector.ts

```diff
@@ -86,6 +87,15 @@ export class IblSelector extends ConnectedLitElement {
             this.config.environmentImage) {
       reduxStore.dispatch(dispatchEnvrionmentImage(
           dropdownElement.selectedItem.getAttribute('value') || undefined));
+      // dropdown value equals null when "Default" is selected
+      if (dropdownElement.selectedItem.getAttribute('value') === null) {
+        reduxStore.dispatch(dispatchSetEnvironmentName(undefined));
+      } else {
+        const envImageList =
+            dropdownElement.selectedItem.getAttribute('value')?.split('/');
```

Comment:
Also, possibly unrelated, but when I select BoomBox, and then load my own model, it gets loaded with the BoomBox textures and appears to just remove the incoming textures completely.

Reply:
fixed